src/index.html

```diff
@@ -13,6 +13,66 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--1">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--2">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--3">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--4">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--5">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line after this `div` element.